target_chains/ethereum/contracts/deploy/zkSyncDeployEntropy.ts

```diff
@@ -90,7 +90,8 @@ export default async function (hre: HardhatRuntimeEnvironment) {
   console.log("Entropy contract address:", entropyContractAddress);
 
   console.log("Saving the contract in the store...");
-  const contract = new EvmEntropyContract(chainName, entropyContractAddress);
+  const chain = DefaultStore.chains[chainName];
+  const contract = new EvmEntropyContract(chain, entropyContractAddress);
```

Comment:
Thank you for adding this. I think we should add wormhole contracts as well to the store.

Reply:
I was too lazy to do it this time. I'll do it next time 😅